manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -35,34 +35,3 @@ spec:
     matchLabels:
       component: apiserver
       provider: kubernetes
----
```

Comment:
@sttts @tnozicka moving this to its own file because i'm not fan of lists

Reply:
Always put moves in their own commits. They are noise.